src/App.jsx

```
@@ -291,964 +142,167 @@ function App() {
-            <h1>VICTORY!</h1>
-            <div className="victory-subtitle">
-              🏆 {selectedCharacter?.name} emerges triumphant!
-            </div>
-            <div className="victory-description">
-              Another opponent falls before your strategic might!
-            </div>
-          </div>
+  if (gameMode === GAME_CONFIG.GAME_MODES.LOCAL) {
+    // Simple local multiplayer mode
+    const { board, currentPlayer, winner, isDraw, winningPieces } = localGameState;
+    const [isMuted, setIsMuted] = useState(false);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)